scripts/run.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+NAME=${NAME:-case-creator}
```

Comment:
Name needs updating

Reply:
That value is set in the Dockefile. I've change the name to direct a future dev who doesn't set it.